src/avfoundation.cs

```diff
@@ -14343,4 +14841,828 @@ public enum AVCaptureCenterStageControlMode : long {
+		AVCaptionDecoration GetDecoration (nint index, [NullAllowed] out NSRange outRange);
+
+		[Export ("textCombineAtIndex:range:")]
+		AVCaptionTextCombine GetTextCombine (nint index, [NullAllowed] out NSRange outRange);
+
+		[Export ("rubyAtIndex:range:")]
+		[return: NullAllowed]
+		AVCaptionRuby GetRuby (nint index, [NullAllowed] out NSRange outRange);
+	}
+
+	[NoWatch, NoTV, NoiOS, Mac (12,0)]
+	[BaseType (typeof(AVCaption))]
+	interface AVMutableCaption
+	{
+		[Export ("initWithText:timeRange:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string text, CMTimeRange timeRange);
+
+		[Export ("text")]
+		string Text { get; set; }
+
+		[Export ("timeRange", ArgumentSemantic.Assign)]
+		CMTimeRange TimeRange { get; set; }
+
+		// interface AVMutableCaption_Styling
+		
+		[Export ("setTextColor:inRange:")]
+		void SetTextColor (CGColor color, NSRange range);
+
+		[Export ("setBackgroundColor:inRange:")]
+		void SetBackgroundColor (CGColor color, NSRange range);
+
+		[Export ("setFontWeight:inRange:")]
+		void SetFontWeight (AVCaptionFontWeight fontWeight, NSRange range);
+
+		[Export ("setFontStyle:inRange:")]
+		void SetFontStyle (AVCaptionFontStyle fontStyle, NSRange range);
+
+		[Export ("setDecoration:inRange:")]
+		void SetDecoration (AVCaptionDecoration decoration, NSRange range);
+
+		[Export ("setTextCombine:inRange:")]
+		void SetTextCombine (AVCaptionTextCombine textCombine, NSRange range);
+
+		[Export ("setRuby:inRange:")]
+		void SetRuby (AVCaptionRuby ruby, NSRange range);
+
+		[Export ("removeTextColorInRange:")]
+		void RemoveTextColor (NSRange range);
+
+		[Export ("removeBackgroundColorInRange:")]
+		void RemoveBackgroundColor (NSRange range);
+
+		[Export ("removeFontWeightInRange:")]
+		void RemoveFontWeight (NSRange range);
+
+		[Export ("removeFontStyleInRange:")]
+		void RemoveFontStyle (NSRange range);
+
+		[Export ("removeDecorationInRange:")]
+		void RemoveDecoration (NSRange range);
+
+		[Export ("removeTextCombineInRange:")]
+		void RemoveTextCombine (NSRange range);
+
+		[Export ("removeRubyInRange:")]
+		void RemoveRuby (NSRange range);
+
+		// interface AVMutableCaption_Region
+		[Export ("region", ArgumentSemantic.Copy)]
+		AVCaptionRegion Region { get; set; }
+
+		[Export ("textAlignment", ArgumentSemantic.Assign)]
+		AVCaptionTextAlignment TextAlignment { get; set; }
+
+		// interface AVMutableCaption_Animation
+		[Export ("animation", ArgumentSemantic.Assign)]
+		AVCaptionAnimation Animation { get; set; }
+	}
+
+	[NoWatch, NoTV, NoiOS, Mac (12,0)]
+	[BaseType (typeof(NSObject))]
+	interface AVCaptionRegion : NSCopying, NSMutableCopying, NSSecureCoding
+	{
+		[Static]
+		[Export ("appleITTTopRegion")]
+		AVCaptionRegion AppleiTTTopRegion { get; }
+
+		[Static]
+		[Export ("appleITTBottomRegion")]
+		AVCaptionRegion AppleiTTBottomRegion { get; }
+
+		[Static]
+		[Export ("appleITTLeftRegion")]
+		AVCaptionRegion AppleiTTLeftRegion { get; }
+
+		[Static]
+		[Export ("appleITTRightRegion")]
```

Comment:
Maybe not anything but, for me, Github shows the color of the code only being black from this point on. Is there perhaps a syntax thing going on somewhere?

Reply:
No, is that the file is too long and probably the js is not dealing with it.